packages/framework-types/src/envelope.ts

```diff
@@ -42,11 +43,7 @@ export interface ReadModelRequestEnvelope extends Envelope {
   filters?: Record<string, ReadModelPropertyFilter>
 }
 
-export interface ReadModelPropertyFilter {
-  operation: string
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  values: Array<any>
-}
+export type ReadModelPropertyFilter = FilterFor<any>
```

Comment:
I'm not sure if it won't give you problems, but Could you please try to change this to `FilterFor<unknown>`? This way we avoid `any` which is a bit dangerous in Typescript.

If you find too many problems after the change, forget it